src/main/scala/com/ing/wbaa/airlock/proxy/handler/RequestHandlerS3.scala

```diff
@@ -1,14 +1,23 @@
 package com.ing.wbaa.airlock.proxy.handler
```

Comment:
I would propose to move the functionality of extracting the request and checking authorization as we do here to a separate class. We can separate concerns a bit more that way and keep this class responsible only for actually handling requests with S3 (not also checking ranger, unfolding request etc.)

Reply:
done